helpers/app

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# Install other YunoHost apps
+#
+# usage: ynh_install_apps --apps="a_ynh b_ynh?domain=DOMAIN&path=PATH&admin=USER&language=fr&is_public=1&password=pass&port=666"
+# | arg: -a, --apps=     - apps to install
+#
+# Requires YunoHost version *.*.* or higher.
+ynh_install_apps() {
+	# Declare an array to define the options of this helper.
+	local legacy_args=a
+	local -A args_array=([a]=apps=)
+	local apps
+	# Manage arguments with getopts
+	ynh_handle_getopts_args "$@"
+
+	# Split the list of apps in an array
+	local apps_list=($(echo $apps | tr " " "\n"))
+	
+	# For each app
+	for i in "${apps_list[@]}"
+	do
+		# Retrieve the name of the app (part before _ynh)
+		local oneapp=$(echo "$i" | awk -F'_ynh' '{print $1}')
+		[ -z "$oneapp" ] && ynh_die --message="You didn't provided a YunoHost app to install"
+
+		# Retrieve the arguments of the app (part after ?)
+		local oneargument=$(echo "$i" | awk -F'?' '{print $2}')
+		[ ! -z "$oneargument" ] && oneargument="--args \"$oneargument\""
+
+		if ! yunohost app list | grep -q "$oneapp"
+		then
+			yunohost tools update
```

Comment:
- `yunohost tools update` is called both in the `if` and `else`, so might as well factorize it outside the if ?
- maybe add `--apps` (or is it `apps` nowadays?) to speed things up and not run the `apt update` which can take significant time on small arm boards

Reply:
done ;)